scripts/build.sh

```diff
@@ -4,25 +4,25 @@
 
 set -e
 
-PROJECT_ROOT=`git rev-parse --show-toplevel`
-source "`dirname \"$0\"`/common.sh"
+PROJECT_ROOT=`git rev-parse --show-toplevel`    # /home/tripleight/code/github.com/paritytech/substrate
+source "`dirname \"$0\"`/common.sh"             # source scripts/common.sh
```

Comment:
This is wrong if `dirname "$0"` ends with a newline, but I don’t think we need to worry about this.

Reply:
I'm about to simplify this.